src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -230,6 +230,12 @@ MessagePtr TwitchMessageBuilder::build()
                 // Actually just text
                 QString linkString = this->matchLink(string);
 
+                if (string[0] == '@' && app->settings->usernameBold) {
```

Comment:
Check if the string has at least one character before accessing the first element.